evm_arithmetization/src/cpu/kernel/aggregator.rs

```diff
@@ -60,8 +62,9 @@ pub static KERNEL_FILES: [&str; NUMBER_KERNEL_FILES] = [
     include_str!("asm/core/precompiles/bn_mul.asm"),
     include_str!("asm/core/precompiles/snarkv.asm"),
     include_str!("asm/core/precompiles/blake2_f.asm"),
+    #[cfg(feature = "eth_mainnet")]
     include_str!("asm/core/precompiles/kzg_peval.asm"),
-    include_str!("asm/curve/bls381/util.asm"),
+    // include_str!("asm/curve/bls381/util.asm"),
```

Comment:
Remove?

Reply:
I'd rather not, as we most likely would need for Pectra upgrade (which includes a bunch of BLS12 precompiles), hence commenting out